### PR TITLE
Proper aspnetcore shutdown

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

**Comment by andreasohlund** [comment]
Update: Seems to be an issue with IISExpress https://stackoverflow.com/a/43844804/236004

Do we keep the code? Switch to Kestrel?

**Comment by andreasohlund** [comment]
Works when selecting kestrel instead of IISExpress

image


Wonder if there is a way to make the the default? cc @bording

**Comment by andreasohlund** [comment]
>Wonder if there is a way to make the the default?

Removing the launchSettings file did the trick

**Comment by DavidBoike** [comment]
I'm 100% OK with this. I don't see the point of keeping IIS Express around when aspnetcore is all about Kestrel.

**Comment by bording** [comment]
Not to mention that kestrel is cross-platform, but IIS Express isn't!

**Comment by andreasohlund** [comment]
This one should be good to go, please review and merge

File: samples/web/asp-mvc-application/Core_7/AsyncPagesMVC.Core/AsyncPagesMVC.Core.csproj [hunk 1]
```diff
@@ -18,4 +18,8 @@
     <ProjectReference Include="..\Shared\Shared.Core.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (bording) [inline samples/web/asp-mvc-application/Core_7/AsyncPagesMVC.Core/AsyncPagesMVC.Core.csproj]: Don't think this should be here

File: samples/web/send-from-aspnetcore-webapi/Core_7/WebApplication/WebApplication.Core.csproj [hunk 2]
```diff
@@ -5,7 +5,10 @@
     <LangVersion>7.1</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <Folder Include="wwwroot\" />
+    <Compile Remove="wwwroot\**" />
```
Comment (bording) [inline samples/web/send-from-aspnetcore-webapi/Core_7/WebApplication/WebApplication.Core.csproj]: Don't think this should be here

File: samples/web/send-from-mvc-controller/Core_7/WebApplication.Core/Startup.cs [hunk 3]
```diff
@@ -18,6 +18,7 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
+
```
Comment (bording) [inline samples/web/send-from-mvc-controller/Core_7/WebApplication.Core/Startup.cs]: 🔥

File: samples/web/send-from-mvc-controller/Core_7/WebApplication.Core/WebApplication.Core.csproj [hunk 4]
```diff
@@ -6,8 +6,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.6" />
-	<PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.2.2" />    
-    <PackageReference Include="NServiceBus.Autofac" Version="7.*" />   
+	<PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.2.2" />
```
Comment (bording) [inline samples/web/send-from-mvc-controller/Core_7/WebApplication.Core/WebApplication.Core.csproj]: Is there a tab here, or is the spacing off?

**Comment by andreasohlund** [comment]
@bording should be good to go now

**Pull request merged**